backend/app_logging/pii_filter.py

```
@@ -0,0 +1,283 @@
+        
+        Args:
+            value: Value to hash
+            
+        Returns:
+            Short hash prefixed with identifier
+        """
+        if not value:
+            return self.redaction_string
+        
+        # Create a short, consistent hash
+        hash_obj = hashlib.md5()
```

Ruff (0.13.1)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)